lib/src/main/java/graphql/nadel/Nadel.kt

```diff
@@ -259,11 +260,17 @@ class Nadel private constructor(
         private var transforms = emptyList<NadelTransform<out Any>>()
         private var introspectionRunnerFactory = NadelIntrospectionRunnerFactory(::NadelDefaultIntrospectionRunner)
 
+        private var schemas: NadelSchemas? = null
         private var schemaBuilder = NadelSchemas.Builder()
 
         private var maxQueryDepth = Integer.MAX_VALUE
         private var maxFieldCount = Integer.MAX_VALUE
 
+        fun schemas(schemas: NadelSchemas): Builder {
```

Comment:
So that our tests can build the schemas separately.

Tbh we probably should remove all the `overallSchema` methods etc. and make the consumer build a `NadelSchemas` themselves and pass it in.